### PR TITLE
Api guide docs

### PR DESCRIPTION
**Pull request by megaSpoon**
## What changes were proposed in this pull request?
api doc

## How was this patch tested?
run gen_site.py

### CODE REVIEW

File: docs/docs/APIGuide/MLPipeline/DLEstimator_DLClassifier.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+      (Array(1.0, 0.0), 2.0),
+      (Array(0.0, 1.0), 1.0),
+      (Array(1.0, 0.0), 2.0)))
+    val df = sqlContext.createDataFrame(data).toDF("features", "label")
+    val dlModel = estimator.fit(df)
+    dlModel.transform(df).show(false)
+  }
+}
+```
+
```
Comment (jason-dai) [inline docs/docs/APIGuide/MLPipeline/DLEstimator_DLClassifier.md]: Add **Python Example:** here

Comment (megaSpoon) [reply docs/docs/APIGuide/MLPipeline/DLEstimator_DLClassifier.md]: fixed

**Comment by megaSpoon** [comment]
I will rebase it after review.

**Comment by jason-dai** [comment]
LGTM

**Comment by megaSpoon** [comment]
same as #1655 rebased and merge to branch-0.3

**Pull request merged**